### PR TITLE
Fixes #3146 and #3144

### LINKED ISSUE #3146: merge field and field_util directories
Not sure of best name.

### PR DESCRIPTION
**Pull request by tclune**
## Types of change(s)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Trivial change (affects only documentation or cleanup)
- [x] Refactor (no functional changes, no api changes)

## Checklist
- [ ] Tested this change with a run of GEOSgcm
- [x] Ran the Unit Tests (`make tests`)

### CODE REVIEW

File: field/FieldCreate.F90 [hunk 1]
```diff
@@ -89,7 +89,7 @@ subroutine field_empty_complete( field, &
 
       bounds = make_bounds(num_levels=num_levels, ungridded_dims=ungridded_dims)
       call ESMF_FieldEmptyComplete(field, typekind=typekind, &
-           gridToFieldMap=gridToFieldMap, &
+!#           gridToFieldMap=gridToFieldMap, &
```
Comment (darianboggs) [inline field/FieldCreate.F90:92]: Delete if not needed.

File: generic3g/specs/FieldSpec.F90 [hunk 2]
```diff
@@ -111,7 +113,7 @@ module mapl3g_FieldSpec
 
       procedure :: make_adapters
 
-      procedure :: set_info
+!#      procedure :: set_info
```
Comment (darianboggs) [inline generic3g/specs/FieldSpec.F90:116]: Delete if unneeded.

**Review by darianboggs — DISMISSED** [review]
There are some commented-out lines you may want to delete.

File: field/FieldCreate.F90 (hunk 1 above)
Comment (tclune) [reply field/FieldCreate.F90:92]: Yeah - not needed now.  Probably will be needed soon.

Comment (tclune) [reply field/FieldCreate.F90:92]: Please do these with the +/- suggestion. Makes it much easier to address.

**Comment by tclune** [comment]
Added trivial changes based upon review.  Skipping CI.

**Pull request merged**